plugin/pkg/admission/noderestriction/admission_test.go

```diff
@@ -53,16 +53,32 @@ var (
 )
 
 func init() {
-	if err := trEnabledFeature.Add(map[featuregate.Feature]featuregate.FeatureSpec{features.TokenRequest: {Default: true}}); err != nil {
+	// all features need to be set on all featuregates for the tests.  We set everything and then then the if's below override it.
+	for _, featureGate := range []featuregate.MutableFeatureGate{
+		trEnabledFeature, trDisabledFeature,
+		csiNodeInfoEnabledFeature, csiNodeInfoDisabledFeature,
+	} {
+		if err := featureGate.Add(map[featuregate.Feature]featuregate.FeatureSpec{features.TokenRequest: {Default: false}}); err != nil {
+			panic(err)
+		}
+		if err := featureGate.Add(map[featuregate.Feature]featuregate.FeatureSpec{features.CSINodeInfo: {Default: false}}); err != nil {
+			panic(err)
+		}
+		if err := featureGate.Add(map[featuregate.Feature]featuregate.FeatureSpec{features.ExpandPersistentVolumes: {Default: false}}); err != nil {
+			panic(err)
+		}
+	}
```

Comment:
the verify script is dumb. I'm sorry. This will work:
```go
	relevantFeatures := map[featuregate.Feature]featuregate.FeatureSpec{
		features.TokenRequest:            {Default: false},
		features.CSINodeInfo:             {Default: false},
		features.ExpandPersistentVolumes: {Default: false},
	}
	utilruntime.Must(trEnabledFeature.Add(relevantFeatures))
	utilruntime.Must(trDisabledFeature.Add(relevantFeatures))
	utilruntime.Must(csiNodeInfoEnabledFeature.Add(relevantFeatures))
	utilruntime.Must(csiNodeInfoDisabledFeature.Add(relevantFeatures))
```

Reply:
> the script that verifies you're not getting a handle to the default mutable feature gate is dumb. I'm sorry. This will work:

Serving our true overlords, the bots.  Problem for monday.

/hold to limit ci retries